src/components/Checkbox/Checkbox.tsx

```diff
@@ -26,12 +26,9 @@ export type CheckboxProps = Omit<CheckboxInputProps, 'id'> & {
 };
 
 /**
- * ```ts
- * import {Checkbox} from "@chanzuckerberg/eds";
- * ```
- * ```ts
- * import {CheckboxInput, CheckboxLabel} from '@chanzuckerberg/eds';
- * ```
+ * `import {Checkbox} from "@chanzuckerberg/eds";`
+ *
```

Comment:
Empty line added to force them to appear on separate lines.

Reply:
I like separate 👍 Distinguishes that you can do either of the options rather than both